README.md

```diff
@@ -150,3 +150,30 @@ When running the app locally, ensure you are using the development `GOOGLE_SHEET
 (See google drive for this)
 
 If updating your environment variable doesn't work, try `rm -rf _build` and then starting your server again
+
+### Image uploads
+
+##### File size
+
+The Wagtail cms allows for a maximum file size of 10MB.
+
+Althouth I would suggest that files be no bigger than 1MB.
+
+Be aware though that many uploads of high resolution images will result in increased costs from our file hosting provider.
+
+##### Resolution
+
+Be aware that if an image has too high a resolution, clients will have to download more onto their device causing pages to take longer to load.
+
+If an image has a too low resolution, then the image could be pixilated on high resolution displays.
+
+##### Preferred file type
+
+Wagtail facilitates GIF, JPEG, PNG, but doesn't specify a preferred file type (you should go for png if you have the choice though)
+
+##### Hero background image
+
+The image should contrasts well with white text on top of it
```

Comment:
You're suddenly missing full stops on these sentences.

Also, it would be good to have a little note with an example of how the image is a bit cut off on mobile (this is standard for a one-size-fits-all hero image) as this doesn't make much difference to a London Eye picture, but will if it's a coffee shop conversation between two people in the foreground for example?